wiki/Beatmap/Circle_size/de.md

```diff
@@ -0,0 +1,39 @@
+---
+tags:
+  - CS
+  - key count
+  - keycount
+---
+
+# Circle Size
+
+*Für Vorschriften zur Circle Size, siehe: [Rankingkriterien](/wiki/Ranking_criteria)*
+
+Die **Circle Size** (***CS***) ist eine Schwierigkeitseinstellung einer [Beatmap](/wiki/Beatmap), die die Größe von [Hit-Objekten](/wiki/Gameplay/Hit_object) beeinflusst. Die Circle Size kann Werte von 0 bis 10 annehmen, der [Beatmapeditor](/wiki/Client/Beatmap_editor) erlaubt aber nur Werte von 2 bis 7. Um andere Werte zu verwenden, muss die [`.osu`-Datei](/wiki/Client/File_formats/osu_(file_format)) der Beatmap manuell bearbeitet werden.
+
+## osu!
+
+In [osu!](/wiki/Game_mode/osu!) bestimmt die Circle Size die Größe der [Hit-Circles](/wiki/Gameplay/Hit_object/Hit_circle) und [Sliders](/wiki/Gameplay/Hit_object/Slider). Je höher der CS-Wert ist, desto kleiner sind die Hit-Objekte. Die [Spinner](/wiki/Gameplay/Hit_object/Spinner)-Größe wird nicht von der Circle Size beeinflusst. Die folgende Formel berechnet die Größe der Hit-Circles:
+
+`r = 54.4 - 4.48 * CS`
```

Comment:
```suggestion
`r = 54,4 - 4,48 * CS`
```

Reply:
I'm not opposed, but this was a deliberate choice because other language also stuck to the decimal dot instead of their native decimal comma.

Reply:
I'll mark this as resolved for now, feel free to comment again if you don't agree @The-Last-Cookie